### PR TITLE
feat(enable-4031): guidelines for scope validation at endpoints level

### PR DESCRIPTION
**Pull request by aryasunny**

### CODE REVIEW

File: chapters/meta-information.adoc [hunk 1]
```diff
@@ -252,3 +252,112 @@ servers:
         default: .dev
 <...>
 ----
+
+[#253]
```
Comment (caliginouspierson) [inline chapters/meta-information.adoc:256]: Nice, I'm pretty happy with this.

Just one note, maybe in your example, mention that an integer **would** be good in this circumstance with another example.

File: chapters/meta-information.adoc [hunk 2]
```diff
@@ -252,3 +252,112 @@ servers:
...
+Basically, Tyk is not able to achieve this
+
+Allow access to **/any-string-value** but not **/status** for the token with a scope **requests:get-request-details**
+
+**Workaround:**
+
+So teams would have to be conscious to not have a string path parameter and another endpoint with a string in it at the same depth.
+
+[source,yaml]
+----
```
Comment (caliginouspierson) [inline chapters/meta-information.adoc:287]: Did you mean to include this? It doesn't seem relevant.

Comment (aryasunny) [reply chapters/meta-information.adoc:287]: yeeaahhh that was un-intentional, will remove it now

File: chapters/meta-information.adoc (hunk 1 above)
Comment (aryasunny) [reply chapters/meta-information.adoc:256]: yeah I thought of that too but then I thought I have mentioned in the description above that it is all about strings
Plus after we merge the other PR, I plan to add location of all the valid and invalid example specification files
And integer being valid is part of one of the unit test scenario..

File: chapters/meta-information.adoc (hunk 2 above)
Comment (aryasunny) [reply chapters/meta-information.adoc]: removed

**Review by caliginouspierson — APPROVED** [review]

**Pull request merged**